DEPENDENCIES.rst

```diff
@@ -55,8 +56,8 @@ Package                Minimum Version
 ====================   ==================
 doxygen (for docs)     1.7.6.1
```

Comment:
(Not necessarily related to this PR) it might be useful to add a column to indicate what feature requires each optional dependency.

Reply:
That is a good idea.  I'll make an issue.